ntcore/src/main/native/cpp/Dispatcher.cpp

```diff
@@ -394,6 +394,25 @@ void DispatcherBase::ServerThreadMain() {
     DEBUG0("server: client connection from " << stream->getPeerIP() << " port "
                                              << stream->getPeerPort());
 
+    char peekBuffer[3];
+    wpi::NetworkStream::Error err;
+    stream->peek(peekBuffer, 3, &err);
+
+    if (!strncmp(peekBuffer, "GET", 3)) {
+      DEBUG0("server: rejecting client from "
+             << stream->getPeerIP() << " port " << stream->getPeerPort()
+             << ". Attempted to connect over websockets.");
+      // Reject websocket connection with non 101 HTTP Response.
+      constexpr auto kResponse =
+          "HTTP/1.1 400 Bad Request\r\n\
+      Connection:close\r\n\
+      Content-Type: text/plain\r\n\
+      \r\n\
+      Server is unable to serve websocket clients. NT4 server may be available on port 5810.";
+      stream->send(kResponse, std::strlen(kResponse), &err);
```

Comment:
Do concatenated strings instead of using \ continuation.  E.g. make each line a string.